tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -3005,6 +3015,10 @@ int main(int argc, char *argv[]) {
         InitInvok.getLangOptions().EffectiveLanguageVersion = actual.getValue();
     }
   }
+  InitInvok.getLangOptions().EnableObjCInterop =
+      llvm::Triple(InitInvok.getTargetTriple()).isOSDarwin()
+          ? !options::DisableObjCInterop
+          : options::EnableObjCInterop;
```

Comment:
Please respect both options in all modes. The target just chooses the default.

Reply:
(I know this has the same effect assuming people don't pass both flags, but it's not a good way to write it.)

Reply:
I believe that this already handles all the cases except the case where both flags are set simultaneously. I can handle it more explicitly though if that's preferred.

Current behavior:
darwin + disable = disabled
darwin + enable = enabled (by virtue of disable being unset and falling back to the default)
non-darwin + enable = enabled
non-darwin + disable = disabled (by virtue of enable being unset and falling back to the default)

Not sure what we'd want to do in the case where somebody sets both (or if we even need to handle that).

Reply:
(disregard my comment, looks like we passed each other while typing)